spec/dummy/config/initializers/openstax_accounts.rb

```diff
@@ -2,18 +2,21 @@
 
 # Initializes a Capybara server running the Dummy app
 CAPYBARA_SERVER = Capybara::Server.new(Rails.application).boot
+Rails.application.config.to_prepare do
 
-OpenStax::Accounts.configure do |config|
-  config.openstax_accounts_url = "http://localhost:#{CAPYBARA_SERVER.port}/"
-  config.openstax_application_id = 'secret'
-  config.openstax_application_secret = 'secret'
-  config.account_user_mapper = User
-  config.logout_via = :delete
-  config.min_search_characters = 3
-  config.max_search_items = 10
+  OpenStax::Accounts.configure do |config|
+    config.openstax_accounts_url = "http://localhost:#{CAPYBARA_SERVER.port}/"
+    config.openstax_application_id = 'secret'
+    config.openstax_application_secret = 'secret'
+    config.account_user_mapper = User
+    config.logout_via = :delete
+    config.min_search_characters = 3
+    config.max_search_items = 10
+
+    # these values copied from the Accounts development env values
```

Comment:
I don't think these are the actual values, thankfully.

Reply:
These weren't changed (aka not my comment), just this whole block was moved into the to_prepare block. But good to know these are safe values to have in here 😆